src/components/diagrams/diagram-pane.tsx

```diff
@@ -1026,7 +1103,8 @@ export function DiagramPane({
     );
 
     const getDiagramTitle = (diagramView: DiagramView) => {
-        return diagramView.svgType !== DiagramType.NETWORK_AREA_DIAGRAM
+        return diagramView.svgType !== DiagramType.NETWORK_AREA_DIAGRAM &&
+            diagramView.svgType !== DiagramType.NAD_FROM_CONFIG
```

Comment:
we make this check at least 10 times in the app, it could be nice to create something like that :
```
export type NAD = DiagramType.NETWORK_AREA_DIAGRAM | DiagramType.NAD_FROM_CONFIG;

function isNadType(type: DiagramType): type is NAD {
    return type === DiagramType.NETWORK_AREA_DIAGRAM || type === DiagramType.NAD_FROM_CONFIG;
}
```

Reply:
Done